test/util/display_SYSINFO_settings.py

```diff
@@ -1,31 +1,28 @@
-# Copyright 2016, EMC, Inc.
+'''
+Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
 
-"""
-  Purpose:
-  This script will generate a list of discovered nodes and display the system information for the nodes 
-"""
+Purpose:
+  This script will generate a list of discovered nodes and display the system information for the nodes
+'''
 
 import fit_path  # NOQA: unused import
-import os
-import sys
-import subprocess
-
 import fit_common
-import test_api_utils
+
+fit_common.VERBOSIY = 1
```

Comment:
These scripts are just reports and the default verbosity makes the reports unreadable. They are not imported or used in any testing.